docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  database:
+    image: 'postgres:latest'
+    container_name: 'fit_database'
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: fit_tracker
+    volumes:
+      - ./db-data/:/var/lib/postgresql/data/
+
+  pgadmin:
+    image: 'dpage/pgadmin4'
+    container_name: 'pgadmin4'
+    ports:
+      - '5050:80'
+    environment:
+      PGADMIN_DEFAULT_EMAIL: user@example.com
+      PGADMIN_DEFAULT_PASSWORD: password
+    volumes:
```

Comment:
Same as above, let's delete it.

Reply:
After deleting pgadmin4 service throws error: 

`You need to define the PGADMIN_DEFAULT_EMAIL and PGADMIN_DEFAULT_PASSWORD or PGADMIN_DEFAULT_PASSWORD_FILE environment variables.`

Need to leave PGADMIN_DEFAULT_EMAIL and PGADMIN_DEFAULT_PASSWORD,
adding: PGADMIN_CONFIG_SERVER_MODE: 'False' will disable login check

Reply:
Same as above